csharp/src/Google.Protobuf.Test/Google.Protobuf.Test.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net462;net6.0</TargetFrameworks>
```

Comment:
How come net462 is removed? I think it's valuable to run tests there. If some features aren't supported on .NET Framework then they can be selectively `#ifdef`ed out.

Reply:
Oops. I removed it to be able to run the test on my unix machine